test/e2e/kubelet_stats.go

```diff
@@ -252,12 +263,70 @@ func getOneTimeResourceUsageOnNode(c *client.Client, nodeName string, cpuInterva
 	return usageMap, nil
 }
 
+func getKubeSystemContainersResourceUsage(c *client.Client) (resourceUsagePerContainer, error) {
+	pods, err := c.Pods("kube-system").List(labels.Everything(), fields.Everything())
+	if err != nil {
+		return resourceUsagePerContainer{}, err
+	}
+	nodes, err := c.Nodes().List(labels.Everything(), fields.Everything())
+	if err != nil {
+		return resourceUsagePerContainer{}, err
+	}
+	nameToContainerIDMap := make(map[string]string)
+	containerIds := make([]string, 0)
+	for _, pod := range pods.Items {
+		for _, container := range pod.Status.ContainerStatuses {
+			nameToContainerIDMap[pod.Name+"/"+container.Name] = strings.TrimPrefix(container.ContainerID, "docker:/")
```

Comment:
containerID := strings.TrimPrefix(container.ContainerID, "docker:/")

and reuse this variable - this would be less error prone and more readable.